OpenMEEG/cmake/UseAuto.cmake

```diff
@@ -0,0 +1,27 @@
+
+if (USE_AUTO)
+    # this macro tries to use library in the defined priority
+    macro(priorities)
+        foreach (lib ${ARGN})
+            string(TOUPPER ${lib} LIB)
+            set(USE_${LIB} true)
+            include(Use${lib})
```

Comment:
I'm slightly worried by the fact that this macro may import some variables from the USEXXX cmake files in the global scope which may create problems difficult to spot. Is there a way to transform this macro into a function and export in the global scope only the variables that we know to need.